Update how-to-create-release-announcements.md (#66)
mattermost/mattermost-handbook: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

         1. **High** - Beautiful, highly compelling, grabs attention in a social stream 
         2. **Medium** - Tells a story clearly, readable text, right size 
         3. **Low** - Bare minimum to call an image 

>    3. **\(Author owns\)** Draft Tweet text for all screenshots included as well as for MVP winner

Reviewer: ```suggestion
   3. **\(Author owns\)** Draft Tweet text for all screenshots and MVP winner included
```
